cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -82,17 +92,14 @@ spec:
               fi;
       # BEGIN_PROMETHEUS_TO_SD
       - name: prometheus-to-sd-exporter
-        image: gcr.io/google-containers/prometheus-to-sd:v0.1.3
+        image: gcr.io/google-containers/prometheus-to-sd:v0.2.1
         command:
           - /monitor
```

Comment:
Now in newer version of prometheus-to-sd it's possible to specify pod name and pod namespace. Here is an example:
https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/prometheus-to-sd/kubernetes/prometheus-to-sd-kube-state-metrics.yaml

Reply:
Ditto